File: mario-more/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int a = 0;
    do
    {
        a = get_int("Height:  ");
    }
    while (a < 1 || a > 8);
    for (int b = a - 1; b > 0; b--)     //for loop for decrementing so spaces can be printed in a descending manner
    {
        for (int c = 0; c < b; c++)     //for loop for incrementing and printing spaces
       
        {
            printf(" ");
            
        }
        for (int d = 0; d < a - b; d++)     //for loop for printing hashes after spaces
        {
            printf("#");
        }
        printf("  ");
        for (int e = 0; e < a - b; e++)
        {
            printf("#");        //for loop for printing next block of hashe
        }
       
        printf("\n");
    }
    for (int f = 0; f < a; f++)
    {
        printf("#");
    }
    printf("  ");
    for (int g = 0; g < a; g++)
    {
        printf("#");
    }
    printf("\n");
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    # central for loop
    for b in range(n - i - 1):
        # loop for space
        print(" ", end="")
    for j in range(i + 1):
        # loop for first set of Hashes
        print("#", end="")
    print("  ", end="")
    # prints Central space
    for k in range(i + 1):
        # loop for last set of Hashes
        print("#", end="")
    print("")